src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentOperations.cs

```diff
@@ -0,0 +1,52 @@
+﻿using SwedbankPay.Sdk.Extensions;
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentOperations : OperationsBase
+    {
+        public TrustlyPaymentOperations(OperationList operations, HttpClient client)
+        {
+            foreach (var httpOperation in operations)
+            {
+                switch (httpOperation.Rel.Value)
+                {
+                    case PaymentResourceOperations.RedirectSale:
+                        RedirectSale = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.ViewSales:
+                        ViewSale = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.UpdatePaymentAbort:
+                        Abort = async payload =>
+                            await client.SendAsJsonAsync<TrustlyPaymentResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateReversal:
+                        Reverse = async payload =>
+                            await client.SendAsJsonAsync<ReversalResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateCancellation:
+                        Cancel = async payload =>
+                            await client.SendAsJsonAsync<CancellationResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+                    default:
```

Comment:
Codacy found an issue: [Remove this empty 'default' clause.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)